bpf/lib/lb.h

```diff
@@ -1316,11 +1352,16 @@ static __always_inline int lb4_local(const void *map, struct __ctx_buff *ctx,
 	__u8 flags = tuple->flags;
 	struct lb4_backend *backend;
 	__u32 backend_id = 0;
+	bool r_skip_l3_xlate = skip_l3_xlate;
```

Comment:
Is there a reason for introducing a new variable here? Can't we just use `skip_l3_xlate`? I don't think it is used somewhere else.